EIPS/eip-6206.md

```diff
@@ -35,8 +35,8 @@ A new instruction, `JUMPF (0xb2)`, is introduced.
 Let the definition of `type[i]` be inherited from [EIP-4750](./eip-4750.md) and define `stack_height` to be the height of the stack at a certain instruction during the instruction flow traversal if the operand stack at the start of the function were equal to `type[i].inputs`.
 
 * The immediate argument of `JUMPF` MUST be less than the total number of code sections.
-* The stack height at `JUMPF` MUST be greater than or equal to `type[code_section_index].inputs`.
-* `type[current_section_index].outputs` MUST equal `stack_height - type[code_section_index].inputs + type[code_section_index].outputs`. This means that `code_section_index` can output less stack elements than the original code section called by the top element on the return stack if the `current_section_index` code section leaves the delta `type[current_section_index].outputs - type[code_section_index].outputs` element(s) on the stack.
+* For each `JUMPF` instruction `type[current_section_index].outputs` MUST be greater or equal `type[code_section_index].outputs`.
```

Comment:
New version:
1. `type[current_section_index].outputs >= type[code_section_index].outputs`
2. `stack_height == type[code_section_index].inputs + type[current_section_index].outputs - type[code_section_index].outputs`

Reply:
I wander if this is not more clear:
```
stack_height == type[current_section_index].outputs + (type[code_section_index].inputs  - type[code_section_index].outputs)
```

Reply:
Maybe slightly. I changed it to this variant (without parentheses)